php/whatsapp-with-vonage/composer.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "templates/starter",
```

Comment:
Let's also update this name to match this template

Reply:
fixed